### ISSUE TITLE
cassandra_object indexing

### ISSUE THREAD

**Issue by romanoff** [NZKoz/cassandra_object#12]
In the first example there is `index :date_of_birth` string. How will this influence Cassandra database? What additional queries will I be able to make?

**Comment by tritonrc** [comment]
Checkout the 'test' directory for use examples.  In this particular example, a column family named 'CustomersByDateOfBirth' is used for the index itself.  Customers.find_all_by_date_of_birth is the only additional API function gained; if the index were designated as unique then Customers.find_by_by_date would be added instead (lib/cassandra_object/indexes.rb)

**Comment by romanoff** [comment]
Thanks a lot. I found test file for indexes. But let's say I have a task and this task has assigned user and status. Both fields are indexed. But I want to get all tasks that are assigned to certain user and that have certain status. As I understand to make this - I should get all indexes for user and make intersection with indexes for certain status. After this find values. Isn't this implemented? Is it far more hard than I wrote? Is it possible to do this?
